File: CSharp/SpaceVIL/SpaceVIL/Items/Frame.cs
using System.Drawing;
using SpaceVIL.Common;

namespace SpaceVIL
{
    public class Frame : Prototype
    {
        static int count = 0;
        public Frame()
        {
            SetItemName("Frame_" + count);
            count++;

            SetStyle(DefaultsService.GetDefaultStyle(typeof(SpaceVIL.Frame)));
            IsFocusable = false;
        }

        internal override bool GetHoverVerification(float xpos, float ypos)
        {
            return false;
        }
    }
}

File: spacevil_21052019/java_src/Frame.java
package com.spvessel.spacevil;

import com.spvessel.spacevil.Common.DefaultsService;

public class Frame extends Prototype {
    private static int count = 0;

    /**
     * Constructs a Frame
     */
    public Frame() {
        setItemName("Frame_" + count);
        count++;
        // setStyle(DefaultsService.getDefaultStyle("SpaceVIL.Frame"));
        setStyle(DefaultsService.getDefaultStyle(Frame.class));
        isFocusable = false;
    }

    @Override
    protected boolean getHoverVerification(float xpos, float ypos) {
        return false;
    }
}